documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Process_Layer/3_4_2_Data_Offering.md

```diff
@@ -44,7 +49,17 @@ To find a Data Provider, the Data Consumer may search in the catalogs of an IDS
 The IDS Metadata Broker then returns the query result to the Data Consumer. The query result may differ depending on the requesting IDS Connector due to filtering of the displayed data according to usage policies defined by the Data Provider. The Data Consumer needs to interpret the result to find out about the different data sources available. Each query result must provide information about each IDS Connector capable of providing the desired data, so the Data Consumer can access each IDS Connector’s Self-Description to learn more about how to receive the desired dataset. The Data Provider may serve the same data using different representations or pricing options, so the Data Consumer may select a suitable offer from the Data Provider.
 
 ![Query Self-Descriptions](media/query-at-broker-activity.png)
-##### Figure 3.4.2.2: Query IDS Metadata Broker
+##### Figure 3.4.2.3: Query IDS Metadata Broker
+
+The IDS Metadata Broker does not serve Vocabularies but provides a reference to a vocabulary and, if required, a reference to a Vocabulary Hub, included in the Self Descption during **Runtime** when a connector is searching for a data provider or a data set. The Data Consumers connector may verify if the data is provided by using a vocabulary that is consumable by the connector, when quering an IDS Metadata Broker or when quering the Self-Description directly from a Data Providers Connector. If the data is not provided in a consumable way, the connector may:
+
+- request the data in a different format
```

Comment:
This could also include the search for other services that could conduct a transformation